server/src/validations/Account.validation.js

```diff
@@ -30,4 +30,34 @@ export default class AccountValidation {
     }
     return next();
   }
+
+  /**
+   *@method validateGetAccount
+
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   *
+   * @return {object} returns status error and message properties
+   */
+  static validateGetAccount(req, res, next) {
+    const { accountNumber } = req.params;
+    const isNum = /^\d+$/; // gotten from Scott Evernden on Stack Overflow
+
+    if (accountNumber.length !== 10) {
```

Comment:
This will return the same error for account numbers that are less than 10 digits. Please differentiate between them.

```suggestion
    if (accountNumber.length > 10) {
```
or change the error message to something like `Account number must be 10 digits`

Reply:
done...